docs/planning/backend/app/services/kaggle_importer.py

```
@@ -0,0 +1,330 @@
+                    # Only update if Kaggle data is more complete
+                    # For now, skip existing
+                    result.skipped += 1
+                else:
+                    FragranceService.create(self.db, fragrance_data)
+                    result.imported += 1
+
+                # Commit in batches
+                if (result.imported + result.updated) % 100 == 0:
+                    self.db.commit()
+
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)